doc/teca_users_guide.tex

```diff
@@ -710,7 +712,7 @@ \subsection{Command Line Arguments}
 \subsection{Example}
 \begin{minted}{bash}
 mpiexec -np 10 ./bin/teca_tc_trajectory_scalars   \
-    --texture ../../TECA_data/earthmap4k.png      \
+    --texture ../../TECA_assets/earthmap4k.png      \
```

Comment:
use the new api in teca_py_config.i to fetch a default internally

Reply:
I don't think we should do that because this is given as a command line argument to the `--texture` option in the `teca_tc_trajectory_scalars` app. We can modify it to `/path/to/background_image.png` if that makes sense. Please let me know if I am missing something.

Reply:
see comments above.